README.md

```diff
@@ -184,4 +186,4 @@ I welcome you to fork and add more features into it. If you have any bugs or fea
 
 ## Leicense
 
-MIT
```

Comment:
now idea why git removed this line just to add it again.
also, notice the typo in the word License above (didn't see it, you probably want to fix it)